core/src/main/java/org/infinispan/health/HealthStatus.java

```diff
@@ -0,0 +1,24 @@
+package org.infinispan.health;
+
+/**
+ * General Health status.
+ *
+ * @author Sebastian Łaskawiec
+ * @since 9.0
+ */
+public enum HealthStatus {
+    /**
+     * Given entity is unhealthy.
+     */
+    RED,
+
+    /**
+     * Given entity is healthy.
+     */
+    GREEN,
+
+    /**
+     * Given entity is healthy but a rebalance is in progress.
+     */
+    YELLOW
```

Comment:
I'd rather call the enum values UNHEALTHY, HEALTHY, REBALANCING, and let the UI decide what color to use.